src/components/pages/project.tsx

```diff
@@ -104,7 +117,8 @@ export const Project = () => {
 								})}
 							</div>
 						</div>
-						<div className="h-full">
+						<div className="flex h-full items-center justify-center">
+							<div className="" id="tourProjectCode" />
```

Comment:
and uncorect popover position
```suggestion
							<div id="tourProjectCode" />
```

Reply:
removed